code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -687,15 +720,16 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "we" = 'sound/vox_fem/we.ogg',
 "weapon" = 'sound/vox_fem/weapon.ogg',
 "welcome" = 'sound/vox_fem/welcome.ogg',
+"weld" = 'sound/vox_fem/weld.ogg',
 "west" = 'sound/vox_fem/west.ogg',
 "whiskey" = 'sound/vox_fem/whiskey.ogg',
 "white" = 'sound/vox_fem/white.ogg',
 "wilco" = 'sound/vox_fem/wilco.ogg',
 "will" = 'sound/vox_fem/will.ogg',
+"window" = 'sound/vox_fem/window.ogg',
+"windows" = 'sound/vox_fem/windows.ogg',
 "with" = 'sound/vox_fem/with.ogg',
 "without" = 'sound/vox_fem/without.ogg',
-"wood" = 'sound/vox_fem/wood.ogg',

```

Comment:
I get "woody" being gone, but this one?

Reply:
Tell me what this'll be used for.

Reply:
"Wood frame detected in arrivals" Is a quick example. I don't think we should be removing stuff like that, instead just punish idiots that try and use it to recite WGW

Reply:
frame isn't a word the AI can say.